docs/source/workflow_inputs.rst

```diff
@@ -192,6 +192,16 @@ A couple schedule CSV file examples are provided in the ``HPXMLtoOpenStudio/reso
 
 A detailed stochastic or smooth schedule CSV file can also be automatically generated for you; see the :ref:`usage_instructions` for the commands.
 
+Generator inputs are entered in ``/HPXML/Building/Site/Address``.
+
+  =================================  ========  =====  ===========  ========  ========  ===============
+  Element                            Type      Units  Constraints  Required  Default   Description
+  =================================  ========  =====  ===========  ========  ========  ===============
+  ``StateCode``                      string                        No        EPW [#]_  Site state code
+  =================================  ========  =====  ===========  ========  ========  ===============
+
+  .. [#] State code will be defined according to the EPW weather file header.
```

Comment:
Please incorporate this into the existing `HPXML Site` section of the documentation.

Reply:
But keep `state_code` and `zip_code` in the Header class, right? (There is `/Building/Site` and there is `Building/BuildingDetails/BuildingSummary/Site`.)

Reply:
Oh, good point, I didn't realize that. In that case, I suggest creating a new "HPXML Building Site" section above (and at the same level as) "HPXML Building Summary". You could also link to that section from here.

Reply:
(Seems like zip code needs to be documented as an optional input too.)

And I'm fine to keep them both in the HPXML Header class for simplicity.